udp_driver/CMakeLists.txt

```diff
@@ -42,11 +50,16 @@ include_directories(include)
 ament_auto_add_library(${PROJECT_NAME} SHARED
```

Comment:
since you are using `ament_auto_add_library` do you need the `include_directories` just above this?